frontend/src/pages/add-quiz-page.tsx

```diff
@@ -0,0 +1,138 @@
+import { Dispatch, SetStateAction, useState } from "react";
+import { Link } from "react-router-dom";
+
+type Question = {
```

Comment:
This type is already exported from `src/types/question.tsx`, we can import from there to stay consistent.

Reply:
You are right. But as  I explained face to face, it makes more sense for me to use this type for add quiz page. Do you think it is possible to use this type definition in our project instead of the Question type you defined earlier?

Reply:
Actually the type I defined also has shortcomings. For example it doesn't have a field to indicate which option is the correct one. We should talk about the exact fields for Question type, implement it in the common type folder and use it from there for consistency.

Reply:
I think Question object should have the followign fiels:
{ 
    id: number ,
    word: string,
    options: string[],
    answer: string,
    type: string
}

What do you think? Are those fields enough?

Reply:
I've added these fields in #298, we can merge this and integrate this page to it with that PR.